docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,27 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+
+## JSON and XDR Conversion Schema
+
+:::info
+
+SDF has published libraries which support the canonical and round-trippable conversion from XDR to JSON, and from JSON to XDR, without loss of information.
+
+:::
+
+The canonical JSON and XDR Schema is defined by the [stellar_xdr crate](https://docs.rs/stellar-xdr/21.1.0/stellar_xdr/index.html) and by the [@stellar/stellar-xdr-json-web npm package](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web).
```

Comment:
```suggestion
A canonical JSON XDR Schema is defined by the [stellar-xdr crate](https://docs.rs/stellar-xdr/21.1.0/stellar_xdr/index.html) and also exposed by the [@stellar/stellar-xdr-json-web npm package](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web).
```

Reply:
It's not obvious from the URL, but the crate name is actually hyphenated and due to how dependencies work in the Rust ecosystem they're always expressed in code with an underscore, but actually the identifier with the hyphen is important because if you try and import the underscore version it won't work.

Reply:
Ah thanks :+1: wasn't sure about the convention